app/Http/Middleware/CORS.php

```diff
@@ -10,10 +10,12 @@ class CORS
 {
     public function handle($request, Closure $next)
     {
+        if (! in_array(env('APP_ENV'), ['local', 'dev'], true)) {
+            return $next($request);
+        }
+
         return $next($request)
-            ->header('Access-Control-Allow-Origin', '*')
-            //->header('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE, OPTIONS');
-            //->header('Access-Control-Allow-Origin', $_SERVER['HTTP_ORIGIN'])
+            ->header('Access-Control-Allow-Origin', $_SERVER['HTTP_ORIGIN'])
```

Comment:
Volgens mij zitten hier de CORS problemen:
1. `env()` calls return null als de config is gecached
2. Lijkt me dat je juist wil dat je op local/dev mode een `Access-Control-Allow-Origin *` wil toevoegen zodat je geen last hebt van CORS

Reply:
Fixed, added env var for origins and updated How to test with it